### ISSUE TITLE
Idea: Translate articles from websites with another language. 

### ISSUE THREAD

**Issue by anon238** [Ranchero-Software/NetNewsWire#3107]
I’m following one website through Safari with the built-in translator. It would be nice to add an option to auto-translate articles.

**Comment by Listen42** [comment]
I agree, having a built-in translation option would be great. I have one or two feeds I follow in another language.

Unfortunately I don't think Apple has made their translation tools available as public APIs yet.

**Comment by HenryKleinschmidt** [comment]
Should be able to natively implement this using Apples Translate Framework: https://developer.apple.com/documentation/translation/
